CMakeLists.txt

```diff
@@ -706,39 +788,49 @@ string(REGEX REPLACE "([^;]+)" "find_dependency(\\1)" _find_dependency_calls "${
 string(REPLACE ";" "\n" _find_dependency_calls "${_find_dependency_calls}")
 
 write_basic_package_version_file(
-	"${CMAKE_CURRENT_BINARY_DIR}/libtorrent-rasterbar/libtorrent-rasterbarConfigVersion.cmake"
+	"${CMAKE_CURRENT_BINARY_DIR}/libtorrent-rasterbar/libtorrent-rasterbar-config-version.cmake"
 	VERSION ${libtorrent_VERSION}
-	COMPATIBILITY ExactVersion
+	COMPATIBILITY SameMajorVersion
```

Comment:
libtorrent breaks ABI compatibility with minor version changes, so this should be `SameMinorVersion`.

Reply:
`SameMinorVersion` appeared in CMake 3.11.

Reply:
Ah ok, that would explain why it was set to `ExactVersion` before. I would rather be too strict than too lenient here so I think it should stay `ExactVersion`.

Reply:
lgtm